README.rst

```diff
@@ -14,8 +14,9 @@ Sentinel-5P Downloader
     :target: https://github.com/emissions-api/sentinel5dl/blob/master/LICENSE
     :alt: MIT license
 
-This library provides easy access to data from the European Space Agency's
-Sentinel 5P satellite.
+sentinel5dl consists of a library and a command line tool which provide easy access to
```

Comment:
Since this is a sentence beginning, I would suggest either capitalize Sentinel5dl or mark it some way as the name of the binary and library, like `sentinel5dl` or __sentinel5dl__.